### PR TITLE
Merge forward from 2021.2 release

### PR DESCRIPTION
**Pull request by mjhilton**
Merges forward https://github.com/OctopusDeploy/Calamari/pull/825, which addresses https://github.com/OctopusDeploy/Issues/issues/7371 for the 2021.3 LTS branch

### CODE REVIEW

File: source/Calamari.Common/Features/StructuredVariables/StructuredConfigVariablesService.cs [hunk 1]
```diff
@@ -3,6 +3,8 @@
 using System.IO;
 using System.Linq;
 using Autofac;
+using Autofac;
```
Comment (mattocto) [inline source/Calamari.Common/Features/StructuredVariables/StructuredConfigVariablesService.cs:6]: You appear to have a duplicated reference here which we could remove.

Comment (mjhilton) [reply source/Calamari.Common/Features/StructuredVariables/StructuredConfigVariablesService.cs:6]: Nice, thankyou!

Comment (mjhilton) [reply source/Calamari.Common/Features/StructuredVariables/StructuredConfigVariablesService.cs]: Fixed

**Review by mattocto — APPROVED** [review]
LGTM!

**Pull request merged**